hw/ip/otbn/dv/otbnsim/sim/ext_regs.py

```diff
@@ -166,7 +178,13 @@ def __init__(self) -> None:
             # reggen's validation should have checked that we have no
             # duplicates.
             assert entry.name not in self.regs
-            self.regs[entry.name] = RGReg(entry)
+            self.regs[entry.name] = RGReg.from_register(entry)
+
+        # Add a fake "StopPC" register. We might well add something like this
+        # to the actual design in the future (see issue #4327) but, for now,
```

Comment:
Could you a TODO marker here for the issue number, otherwise I'm sure we'll never find this reference again.